packages/apps/shopify-app-remix/src/server/authenticate/admin/authenticate.ts

```diff
@@ -140,11 +140,6 @@ export function authStrategyFactory<
         shop,
       });
 
-      logger.debug('Request is valid, loaded session from session token', {
```

Comment:
In each Auth strategy we are logging if the request is valid. 
I don't think we need this as we aren't always loading session from session token?

Reply:
seems like we throw an exception in case the auth process fails, so I agree we don't need this log

Reply:
No objections here!